lighthouse-core/config/config.js

```diff
@@ -395,40 +383,59 @@ class Config {
   }
 
   /**
-   * @param {LH.Config} config
-   * @return {LH.Config}
+   * @param {LH.Config.Json['passes']} passes
+   * @return {?Array<Required<LH.Config.PassJson>>}
    */
-  static augmentWithDefaults(config) {
-    const {defaultSettings, defaultPassConfig} = constants;
-    config.settings = merge(deepClone(defaultSettings), config.settings, true);
-    if (config.passes) {
-      config.passes = config.passes.map(pass => merge(deepClone(defaultPassConfig), pass));
+  static augmentPassesWithDefaults(passes) {
+    if (!passes) {
+      return null;
     }
 
-    return config;
+    const {defaultPassConfig} = constants;
+    return passes.map(pass => merge(deepClone(defaultPassConfig), pass));
```

Comment:
~~I think I may have missed this last time, but don't we need safe deepClone here for `.gatherers` ala deepCloneConfigJson?~~

nevermind we only clone the defaults, right? 👍

Reply:
yeah, the default is an empty `gatherers` array so it's fine, though I guess it could be fragile if we ever change that in the future...